### PR TITLE
OPENEUROPA-1340: Add a custom command in the Runner configuration to update permissions automatically.

### PR DESCRIPTION
**Pull request by drupol**

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -60,17 +61,16 @@ pipeline:
     group: test
     image: fpfis/httpd-php-dev:7.1
     commands:
-      - ./vendor/bin/phpunit
+      - chmod -R 777 /tmp/html/build/sites -R
```
Comment (pfrenssen) [inline .drone.yml]: Instead of doing this blanket change of making every single file in the site writeable and executable, can we maybe try to set it up properly?

Maybe something like this would be more appropriate:

```
- chown -R www-data /tmp/html/build/sites
- find /tmp/html/build/sites/default/files -type d -exec chmod 775 {} \;
- find /tmp/html/build/sites/default/files -type f -exec chmod 664 {} \;
```

I'm assuming here that only the files folder should be writable at this point, but I might be wrong in this. It is my understanding that the files in `sites/` should not be writable but those in `sites/default/files` should be.

File: .drone.yml [hunk 2]
```diff
@@ -60,17 +61,16 @@ pipeline:
...
   behat:
     group: test
     image: fpfis/httpd-php-dev:7.1
     commands:
-      # Workaround for fixing problem with files permission inside docker
-      # @todo remove first 2 lines after docker image release which contain fix of this problem
-      - chmod -R 777 build/sites/default/files
-      - ./vendor/bin/drush cr
-      - ./vendor/bin/behat
+      - chmod -R 777 /tmp/html/build/sites -R
```
Comment (pfrenssen) [inline .drone.yml]: Same here, I would try to set up realistic file system permissions instead of making everything world writable and executable. Files in the `sites/default/files` folder should not be executable.

File: tests/FunctionalJavascript/MediaDocumentTest.php [hunk 3]
```diff
@@ -31,11 +31,10 @@ public function setUp() {
       'create document media',
     ]);
 
-    $this->drupalLogin($editor);
+    // This first drupalGet() is needed.
```
Comment (pfrenssen) [inline tests/FunctionalJavascript/MediaDocumentTest.php]: Why is this needed? Please elaborate :)

File: tests/FunctionalJavascript/MediaEntityBrowserTest.php [hunk 4]
```diff
@@ -35,6 +35,9 @@ public function setUp() {
       'access media_entity_browser entity browser pages',
     ]);
 
+    // This first drupalGet() is needed.
```
Comment (pfrenssen) [inline tests/FunctionalJavascript/MediaEntityBrowserTest.php]: Same here, please explain the "why" and not the "what" ;)

File: tests/FunctionalJavascript/MediaImageTest.php [hunk 5]
```diff
@@ -32,11 +32,10 @@ public function setUp() {
       'access media_entity_browser entity browser pages',
     ]);
 
-    $this->drupalLogin($editor);
+    // This first drupalGet() is needed.
```
Comment (pfrenssen) [inline tests/FunctionalJavascript/MediaImageTest.php]: Here too.

File: tests/FunctionalJavascript/MediaRemoteVideoTest.php [hunk 6]
```diff
@@ -52,6 +52,9 @@ public function setUp() {
       'access media_entity_browser entity browser pages',
     ]);
 
+    // This first drupalGet() is needed.
```
Comment (pfrenssen) [inline tests/FunctionalJavascript/MediaRemoteVideoTest.php]: Another instance.

File: tests/FunctionalJavascript/MediaDocumentTest.php (hunk 3 above)
Comment (drupol) [reply tests/FunctionalJavascript/MediaDocumentTest.php]: To set the session cookie properly.

File: .drone.yml (hunk 1 above)
Comment (drupol) [reply .drone.yml]: You're perfectly right, I will submit the changes and rebase the branch properly.

**Comment by pfrenssen** [comment]
It looks good, but can you please still explain why the drupalGet() is needed? Maybe something like

```
// Ensure that a Mink session is available. A session is initialized when visiting a page.
```

**Comment by drupol** [comment]
Tests are passing, so I guess that comment is no more needed ?
@pfrenssen you confirm ?

**Comment by upchuk** [comment]
@drupol @pfrenssen Approved and merged. We have a followup to remove all the functional tests and replace with behat but we will need these permissions fixes for that as well. Plus the readme is good.

Thanks!
